x-pack/plugins/global_search_bar/public/search_syntax/parse_search_params.ts

```diff
@@ -16,12 +16,40 @@ const aliasMap = {
   type: ['types'],
 };
 
-export const parseSearchParams = (term: string): ParsedSearchParams => {
+// Converts multi word types to phrases by wrapping them in quotes. Example: type:canvas workpad -> type:"canvas workpad"
+const convertMultiwordTypesToPhrases = (term: string, multiWordTypes: string[]): string => {
+  if (multiWordTypes.length === 0) {
+    return term;
+  }
+
+  const typesPattern = multiWordTypes.join('|');
+  const canvasWorkpadRegex = new RegExp(`(type:|types:)(\\s*[^"']*?)\\b(${typesPattern})\\b`, 'gi');
```

Comment:
nit: can we use a generic name, e.g. `termReplaceRegex`?

Reply:
Oops, thanks for noticing this. This variable name is something I used initially, before I made this solution more generic. I'll change that.